### PR TITLE
MAINT: minor fixes to setup.cfg and MANIFEST

### LINKED ISSUE #302: MNT: Clean up MANIFEST.in
A lot of stuff copied over from template and not relevant here.

This is a follow-up of #290 .

### PR DESCRIPTION
**Pull request by bsipocz**
This will close #302 and also clears up the inconsistency between code and docs line length.

### CODE REVIEW

**Comment by bsipocz** [comment]
There is only one expected linter error due to #378. I expect the fix for that bug is trivial and quick, so I didn't apply any workarounds for it in this set of linter fixes.

File: pyvo/registry/tests/test_rtcons.py [hunk 1]
```diff
@@ -67,40 +67,44 @@ def test_odd_type_rejected(self):
...
+            "UNION SELECT ivoid FROM rr.resource WHERE 1=ivo_hasword(res_title, 'star') "
+            "UNION SELECT ivoid FROM rr.res_subject WHERE res_subject ILIKE '%star%')")
 
     def test_interesting_literal(self):
-        assert (rtcons.Freetext("α Cen's planets").get_search_condition()
-            == "ivoid IN (SELECT ivoid FROM rr.resource WHERE 1=ivo_hasword(res_description, 'α Cen''s planets') UNION SELECT ivoid FROM rr.resource WHERE 1=ivo_hasword(res_title, 'α Cen''s planets') UNION SELECT ivoid FROM rr.res_subject WHERE res_subject ILIKE '%α Cen''s planets%')")
+        assert rtcons.Freetext("α Cen's planets").get_search_condition() == (
+            "ivoid IN (SELECT ivoid FROM rr.resource WHERE 1=ivo_hasword(res_description, 'α Cen''s planets')"
+            " UNION SELECT ivoid FROM rr.resource WHERE 1=ivo_hasword(res_title, 'α Cen''s planets') "
+            "UNION SELECT ivoid FROM rr.res_subject WHERE res_subject ILIKE '%α Cen''s planets%')")
```
Comment (andamian) [inline pyvo/registry/tests/test_rtcons.py:79]: nitpicking - consistency in whitespaces in front of 'UNION`

File: pyvo/registry/tests/test_rtcons.py [hunk 2]
```diff
@@ -222,98 +221,93 @@ class TestSpectralConstraint:
...
 
 
 class TestWhereClauseBuilding:
     @staticmethod
     def where_clause_for(*args, **kwargs):
-        cons = list(args)+rtcons.keywords_to_constraints(kwargs)
+        cons = list(args) + rtcons.keywords_to_constraints(kwargs)
         return rtcons.build_regtap_query(cons
-            ).split("\nWHERE\n", 1)[1].split("\nGROUP BY\n")[0]
+                                         ).split("\nWHERE\n", 1)[1].split("\nGROUP BY\n")[0]
```
Comment (andamian) [inline pyvo/registry/tests/test_rtcons.py:286]: The new line seems mispositioned IMO.

**Review by andamian — APPROVED** [review]
I hope you've used a reformat tool... Thank you!

**Comment by bsipocz** [comment]
> I hope you've used a reformat tool... Thank you!

yep, used autopep8 which took care of most of the things, and then I touched up the remaining ones manually.

**Pull request merged**